Brooks/LanguageRenderer/NewCSharpRenderer.hs

```diff
@@ -0,0 +1,561 @@
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
Is the option actually needed?

Reply:
Yes, for type declarations within typeclasses.